CRM/Utils/Check/Component/Schema.php

```diff
@@ -145,34 +145,28 @@ public function checkSmartGroupCustomFieldCriteria() {
               'sequential' => 1,
               'id' => $field['cfid'],
-            ]);
-            $fieldName = ts('<a href="%1" title="Edit Custom Field"> %2 </a>', [
-              1 => CRM_Utils_System::url('civicrm/admin/custom/group/field/update',
-                "action=update&reset=1&gid={$customField['custom_group_id']}&id={$field['cfid']}", TRUE
-              ),
-              2 => $customField['label'],
-            ]);
+           ]);
+            $url = CRM_Utils_System::url('civicrm/admin/custom/group/field/update', "action=update&reset=1&gid={$customField['custom_group_id']}&id={$field['cfid']}", TRUE);
+            $fieldName = '<a href="' . $url . '" title="' . ts('Edit Custom Field', ['escape' => 'js']) . '">' . $customField['label'] . '</a>';
           }
           catch (CiviCRM_API3_Exception $e) {
-            $fieldName = ' <span style="color:red"> - Deleted - </span> ';
+            $fieldName = '<span style="color:red">' . ts('Deleted') . ' - ' . ts('Field ID %1', [1 => $field['cfid']]) . '</span> ';
           }
         }
-        $groupEdit = '<a href="' . CRM_Utils_System::url('civicrm/contact/search/advanced', "?reset=1&ssID={$field['ssid']}", TRUE) . '" title="' . ts('Edit search criteria') . '"> <i class="crm-i fa-pencil" aria-hidden="true"></i> </a>';
-        $groupConfig = '<a href="' . CRM_Utils_System::url('civicrm/group', "?reset=1&action=update&id={$id}", TRUE) . '" title="' . ts('Group settings') . '"> <i class="crm-i fa-gear" aria-hidden="true"></i> </a>';
+        $groupEdit = '<a href="' . CRM_Utils_System::url('civicrm/contact/search/advanced', "?reset=1&ssID={$field['ssid']}", TRUE) . '" title="' . ts('Edit search criteria', ['escape' => 'js']) . '"> <i class="crm-i fa-pencil" aria-hidden="true"></i> </a>';
+        $groupConfig = '<a href="' . CRM_Utils_System::url('civicrm/group', "?reset=1&action=update&id={$id}", TRUE) . '" title="' . ts('Group settings', ['escape' => 'js']) . '"> <i class="crm-i fa-gear" aria-hidden="true"></i> </a>';
         $html .= "<tr><td>{$id} - {$field['title']} </td><td>{$groupEdit} {$groupConfig}</td><td class='disabled'>{$fieldName}</td>";
       }
 
-      $message = "<p>The following smart groups include custom fields which are disabled/deleted from the database. This may cause errors on group page.
-        You might need to edit their search criteria and update them to clean outdated fields from saved search OR disable them in order to fix the error.</p>
-        <p><table><thead><tr><th>Group</th><th></th><th>Custom Field</th>
-        </tr></thead><tbody>
-        $html
-        </tbody></table></p>
-       ";
+      $message = "<p>" . ts('The following smart groups include custom fields which are disabled/deleted from the database. This may cause errors on group page. You might need to edit their search criteria and update them to clean outdated fields from saved search OR disable them in order to fix the error.') . '</p>'
```

Comment:
This was also there before but I'm not clear on what "errors on group page" means. Maybe it should be "This may cause errors when using the group." At the very least it should have a "the" in it, but I don't know what "the group page" is referring to.

Reply:
I updated for now to "This may cause errors when recalculating the smart group counts", because I think that's the most 
accurate problem, and it's also the name of the button on the "Manage Groups" page (I'd refer to the name of that section explicitly, but I couldn't find a non-awkward way to say it).

but will circle back